lib/skylight/user_config.rb

```diff
@@ -14,7 +14,19 @@ def initialize
     end
 
     def file_path
-      File.expand_path(ENV["SKYLIGHT_USER_CONFIG_PATH"] || "~/.skylight")
+      return @file_path if @file_path
```

Comment:
@wagenet better IMHO to `return @file_path if defined?(@file_path)` else you'll get a Ruby warning for an uninitialized ivar